src/main/java/gg/solarmc/clans/command/commands/ally/RemoveCommand.java

```diff
@@ -22,18 +31,27 @@ public void execute(CommandSender sender, String[] args, PluginHelper helper) {
             return;
         }
 
-        if(!helper.isLeader(clan, player)){
+        if (!helper.isLeader(clan, player)) {
             player.sendMessage(Component.text("Only Clan Leader can use this Command", NamedTextColor.RED));
             return;
         }
 
         Component confirmMsg = Component.text("Confirm Message : Use ", NamedTextColor.YELLOW)
-                .append(Component.text("/clan allyremove confirm", NamedTextColor.GOLD))
-                .append(Component.text(" to remove the Ally :)",NamedTextColor.YELLOW));
+                .append(Component.text("/ally remove confirm", NamedTextColor.GOLD))
+                .append(Component.text(" to remove the Ally :)", NamedTextColor.YELLOW))
+                .append(Component.newline())
+                .append(Component.text("Click to Confirm")
+                        .clickEvent(ClickEvent.runCommand("clan disband confirm")));
 
         if (helper.invalidateConfirm(player, args, confirmMsg, 0)) return;
 
-        player.getServer().getDataCenter().runTransact(clan::revokeAlly);
+        player.getServer().getDataCenter().runTransact(clan::revokeAlly)
```

Comment:
revokeAlly returns a boolean which you need to check

Reply:
So i'll need to check this for all commands , right ? or only this u want